arangod/RocksDBEngine/Listeners/RocksDBThrottle.cpp

```diff
@@ -333,14 +333,14 @@ void RocksDBThrottle::RecalculateThrottle() {
       int64_t temp_rate = _throttleBps.load(std::memory_order_relaxed);
 
       if (temp_rate < new_throttle) {
-        temp_rate += (new_throttle - temp_rate) / _scalingFactor + 1;
+        temp_rate += (new_throttle - temp_rate) / _scalingFactor;
```

Comment:
Now `temp_rate` can go down to as far as 0, and previously it could go down to at least 1.
`temp_rate` is used as an input for `_throttleBps`, which is compared to a minimum value of 100 in line 394 anyway.
So what is the net effect of the changes in this PR?